src/components/TopBar/README.md

```diff
@@ -370,6 +372,7 @@ function TopBarExample() {
       onChange={handleSearchChange}
       value={searchValue}
       placeholder="Search"
+      showFocusBorder
```

Comment:
To make it easier for the plus team, I updated the example to use their white theme.